sdk/storage/sample/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.12)
+
+set(TARGET_NAME "azure-storage-sample")
+
+project (${TARGET_NAME} LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_executable (
+     ${TARGET_NAME}
+     blob_getting_started.cpp 
+)
+
+target_link_libraries(${TARGET_NAME} PRIVATE azure-core azure-storage)
```

Comment:
sample executable should be agnostic about `azure-core`. Think sample as our customers' executable, it should only depend on azure-storage library.